src/Keboola/DbExtractor/Extractor/Redshift.php

```diff
@@ -51,7 +53,9 @@ public function testConnection(): void
     public function getTables(?array $tables = null): array
     {
         $sql = "SELECT * FROM information_schema.tables 
-                WHERE table_schema != 'pg_catalog' AND table_schema != 'information_schema'";
+                WHERE table_schema != 'pg_catalog' 
+                    AND table_schema != 'information_schema' 
+                    AND table_schema != 'pg_internal'";
```

Comment:
tohle je novy, proc?

Reply:
bez toho se objevují i tabulky z `pg_internal` -> je to i na masteru a ty tabulky jsou pro getTables extractoru zbytečný

Reply:
a kvůli tomu neprojdou testy getTables